packages/cli/src/evaluation.ee/test-runner/__tests__/create-pin-data.ee.test.ts

```diff
@@ -1,104 +0,0 @@
-import { readFileSync } from 'fs';
-import path from 'path';
-
-import { createPinData } from '../utils.ee';
-
-const wfUnderTestJson = JSON.parse(
-	readFileSync(path.join(__dirname, './mock-data/workflow.under-test.json'), { encoding: 'utf-8' }),
-);
-
-const wfUnderTestRenamedNodesJson = JSON.parse(
-	readFileSync(path.join(__dirname, './mock-data/workflow.under-test-renamed-nodes.json'), {
-		encoding: 'utf-8',
-	}),
-);
-
-const executionDataJson = JSON.parse(
-	readFileSync(path.join(__dirname, './mock-data/execution-data.json'), { encoding: 'utf-8' }),
-);
-
-describe('createPinData', () => {
-	test('should create pin data from past execution data', () => {
-		const mockedNodes = [
-			{
-				id: '72256d90-3a67-4e29-b032-47df4e5768af',
-				name: 'When clicking ‘Execute workflow’',
-			},
-		];
-
-		const pinData = createPinData(wfUnderTestJson, mockedNodes, executionDataJson);
```

Comment:
why are we deleting these without deleting actual function `createPinData` in utils?

Reply:
`createPinData` is not used anymore, deleted `utils` completely